util.cpp

```diff
@@ -141,8 +141,7 @@ void Util::keyReleaseFeedback()
     if( !settingsValue("ui/keyPressFeedback").toBool() )
         return;
 
-    // TODO: check what's more comfortable, only press, or press and release
-    QFeedbackEffect::playThemeEffect(QFeedbackEffect::ReleaseWeak);
+    // No haptic release feedback
```

Comment:
Is it easy to change the QFeedback effect on SbJ for ReleaseWeak? Is that in the ngfd area?